test/cpp/end2end/grpclb_end2end_test.cc

```diff
@@ -536,10 +534,10 @@ class GrpclbEnd2endTest : public ::testing::Test {
     return status;
   }
 
-  void CheckRpcSendOk(const size_t times = 1) {
+  void CheckRpcSendOk(size_t times = 1, int timeout_ms = 1000) {
```

Comment:
`times` should still be const? So does `timeout_ms`.

Reply:
Making const a variable passed by value makes no difference.

Reply:
BUT, well, it'd be tantamount to declaring any other variable local to the function as `const`, and I'm usually the one preaching about it, so yeah, let me make them both const.